wom/src/main/scala/wdl4s/wdl/Declaration.scala

```diff
@@ -80,6 +84,8 @@ trait DeclarationInterface extends WdlGraphNodeWithUpstreamReferences {
 
 object Declaration {
 
+  type DeclarationNode = Either[ExpressionNode, GraphInputNode]
```

Comment:
FWIW I can't shake the feeling that `Either` represents a right or wrong case.  

I know the essence is disjunction, but whenever I see Left I think bad.  The right-biased-ness of it doesn't help.